tests/tests.rs

```diff
@@ -1,28 +1,30 @@
 use pearl::Builder;
 
 #[test]
-fn test_storage_init() {
-    let storage_builder = Builder::new().work_dir("/tmp/");
-    let _storage = storage_builder.build();
+fn test_storage_init_new() {
+    let storage_builder = Builder::new().work_dir("/tmp/pearl/");
+    let mut storage = storage_builder.build();
+    assert!(storage.init().map_err(|e| eprintln!("{}", e)).is_ok());
```

Comment:
It is not good to use asserts in both debug and release modes. In release mode you should handle all error situations gracefully and of course make logs about it. For debug stage of development you better use [debug asserts](https://doc.rust-lang.org/std/macro.debug_assert.html) that will not be included in code in release mode. But I recommend you to not use runtime asserts at all and use only [static asserts](https://crates.io/crates/static_assertions) which works only in compile time.

Reply:
Oh, I didn't notice, that it is tests, sorry :)

Reply:
Thanks for the review) 
In addition, Rust `assert`s rarely used in safe code, in general in unsafe code and tests.